src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -26,9 +26,8 @@ internal class FormattingVisitor : SyntaxWalker
         private int _currentComponentIndentationLevel = 0;
         private bool _isInClassBody = false;
 
-        public FormattingVisitor(RazorSourceDocument source!!)
+        public FormattingVisitor()
         {
-            _source = source;
```

Comment:
Ahhh so this was extra and we never used it?

Reply:
Yeah, just unused